.github/labeler.yml

```diff
@@ -0,0 +1,87 @@
+api:
+  - "app/bundles/ApiBundle/**"
+  
+assets:
+  - "app/bundles/AssetBundle/**"
+  
+campaigns:
+  - "app/bundles/CampaignBundle/**"
+  
+categories:
+  - "app/bundles/CategoryBundle/**"
+  
+channels:
+  - "app/bundles/ChannelBundle/**"
+  
+companies:
+  - "**/*company*"
+  
+configuration:
+  - "app/bundles/ConfigBundle/**"
+  
+dashboard:
+  - "app/bundles/DashboardBundle/**"
+  
+dynamic content:
+  - "app/bundles/DynamicContentBundle/**"
+  
+email:
+  - "app/bundles/EmailBundle/**"
+  
+forms:
+  - "app/bundles/FormBundle/**"
+  
+import-export:
+  - "**/*import*"
+  
+installation:
+  - "app/bundles/InstallBundle/**"
+  
+integrations:
+  - "app/bundles/IntegrationsBundle/**"
+  
+landing pages:
+  - "app/bundles/PageBundle/**"
+  
+marketplace:
+  - "app/bundles/MarketplaceBundle/**"
+  
+notifications:
+  - "app/bundles/NotificationBundle/**"
+  
+plugin:
+  - "app/bundles/PluginBundle/**"
+  
+points-scoring:
+  - "app/bundles/PointBundle/**"
+  
+queue:
+  - "app/bundles/QueueBundle/**"
+  
+reports:
+  - "app/bundles/ReportBundle/**"
+  
+roles:
+  - "app/bundles/UserBundle/**"
```

Comment:
A change in the UserBundle doesn't necessarily mean it is a roles change. Roles are just a part of the user bundle/

Reply:
I think we use the Roles label both for roles and changes related to users (as we don’t have a users label)

Is that what you’re talking about?

If not, how could we improve?